sql/firefox_nondesktop_exact_mau28_raw_v1.sql

```diff
@@ -22,28 +22,33 @@ SELECT
     WHEN 'Zerda' THEN 'Firefox Lite'
     ELSE app_name
   END AS product,
+  normalized_channel,
   campaign,
   country,
   distribution_id
 FROM
   inactive_days
 WHERE
-  -- This list corresponds to the products considered for 2019 nondesktop KPIs.
+  -- This list corresponds to the products considered for 2019 nondesktop KPIs;
+  -- we apply this filter here rather than in the live view because this field
+  -- is not normalized and there are many single pings that come in with unique
+  -- nonsensical app_name values.
   app_name IN (
     'Fennec', -- Firefox for Android and Firefox for iOS
     'Focus',
     'Zerda', -- Firefox Lite, previously called Rocket
     'FirefoxForFireTV', -- Amazon Fire TV
     'FirefoxConnect' -- Amazon Echo Show
     )
+  -- There are also many strange nonsensical entries for os, so we filter here.
   AND os IN ('Android', 'iOS')
-  AND normalized_channel = 'release'
   -- 2017-01-01 is the first populated day of telemetry_core_parquet, so start 28 days later.
-  AND @submission_date >= '2017-01-28'
-  AND @submission_date = submission_date
+  AND submission_date >= DATE('2017-01-28')
+  -- AND @submission_date = submission_date
```

Comment:
I don't think we need to reference the query parameter in the view, even as a comment
```suggestion
```

Reply:
Yeah, I don't think I intended to commit this.